packages/mobile/README.md

```diff
@@ -369,8 +369,9 @@ See [`src/identity/verification.test.ts`] for an example.
 We use [Detox][detox] for E2E testing. In order to run the tests locally, you
 must have the proper emulator set up. Follow the instructions in [e2e/README.md][e2e readme].
 
-Once setup is done, you can run the tests with `yarn test:e2e:android` or `yarn test:e2e:ios`.
-If you want to run a single e2e test: `yarn test:e2e:ios -f Exchange.spec.js -t "Then Buy CELO"`
+Once setup is done, you can buil the tests with `yarn e2e:build:android-release` or `yarn e2e:build:ios-release`.
```

Comment:
`build` :D

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2381/commits/92a6d9afafabfbd72bb588ff4cd66ffec4fce54d!